WalletWasabi.Backend/Controllers/WabiSabiController.cs

```diff
@@ -152,4 +157,19 @@ public HumanMonitorResponse GetHumanMonitor()
 
 		return new HumanMonitorResponse(response.ToArray());
 	}
+
+	/// <summary>
+	/// Gets the list of unconfirmed coinjoin transaction Ids.
+	/// </summary>
+	/// <returns>The list of coinjoin transactions in the mempool.</returns>
+	/// <response code="200">An array of transaction Ids</response>
+	[HttpGet("unconfirmed-coinjoins")]
+	[ProducesResponseType(200)]
+	public IActionResult GetUnconfirmedCoinjoins()
```

Comment:
What's the idea with this endpoint? Are we going to use this one? the one on `BatchController`? both?

Reply:
Same as above https://github.com/zkSNACKs/WalletWasabi/pull/10956#discussion_r1247550914

Reply:
I think it is a useful endpoint. Good for testing purposes in case of a mempool spike.